usr/bin/update-torbrowser

```diff
@@ -88,7 +88,7 @@ tb_error_handler() {
    [ -n "$tb_install_folder" ] || tb_install_folder="tb"
 
    mkdir --parents "$tb_user_home/.cache/$tb_install_folder"
-   echo "$MSG" >> "$tb_user_home/.cache/$tb_install_folder/torbrowser_updater_error.log"
+   echo "$MSG" >> "$tb_user_home/.cache/$tb_install_folder/$tb_bin.updater_error.log"
 
```

Comment:
Option a:
Keep file name. Currently `$tb_bin.` changes it (Do `.` rather than underscore `_`.)

----

Option b:
Change file name but then update where the same file name is being used.

* tb-updater
* tb-starter
* whonixcheck

( Since there `$tb_install_folder` already is a variable that differs, we could simplify. (`"$tb_user_home/.cache/$tb_install_folder/tpo_updater_error.log"`) )